src/ffi_export.rs

```diff
@@ -270,9 +270,11 @@ macro_rules! __ffi_export__ {(
                             b"/** \\brief\n",
                         )?;
                         $(
-                            $crate::core::write!(out,
-                                " * {}\n", $doc,
-                            )?;
+                            $crate::core::write!(out, " *")?;
+                            if !$doc.is_empty() {
+                                $crate::core::write!(out, " {}", $doc)?;
+                            }
+                            $crate::core::write!(out, "\n")?;
```

Comment:
Another implementation could be to:

```suggestion
                            $crate::core::write!(out,
                                " *{sep}{}\n", $doc, sep = if $doc.is_empty() { "" } else { " " },
                            )?;
```

It's not necessarily better, but it's already quite present in the current codebase of `safer-ffi`, so it may be better?

Reply:
Yeah I like that better. Will update 👍